tailwind.config.js

```diff
@@ -5,7 +5,13 @@ module.exports = {
   "./src/**/*.{js,jsx,ts,tsx}",
   ],
   theme: {
-  extend: {},
+    extend: {
+      fontFamily: {
+        header: ['Fjalla One', 'sans-serif'],
+        body: ['Merriweather Sans', 'sans-serif'],
+        button: ['Merriweather Sans', 'sans-serif'],
```

Comment:
```suggestion
        'sans': ['Merriweather Sans', ...defaultTheme.fontFamily.sans]
```
nitpick: It's not wrong to do this, since it will allow us to have a `font-body` class, but it's unnecessary. Instead, I'd suggest overriding the default `sans` (which TW applies to our HTML by default).
suggestion(nb): Read more about setting [default fonts in TW](https://tailwindcss.com/docs/font-family#customizing-the-default-font).